src/client/components/MediaPlayer/index.js

```diff
@@ -1,29 +1,78 @@
-import React from 'react';
+import React, { Component } from 'react';
 import ReactJWPlayer from 'react-jw-player';
 import PropTypes from 'prop-types';
+import axios from 'axios';
 
-export const MediaPlayer = ({ type, mediaURL }) => {
-  MediaPlayer.propTypes = {
-    mediaURL: PropTypes.string,
-    type: PropTypes.string,
-  };
-  const onPlayerError = e => {
-    console.log(e);
-  };
+import { ltikPromise } from '../../services/ltik';
 
-  let imageURL = '';
-  if (type === 'audio' || type === 'a') {
-    imageURL =
-      'https://www.shutupandtakemymoney.com/wp-content/uploads/2020/04/zoom-meeting-audio-vs-video-meme.jpg';
+const constants = require('../../../../constants');
+
+export class MediaPlayer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { playbackPos: 0 };
+  }
+
+  shouldComponentUpdate(nextProps, nextState) {
+    const { media } = this.props;
+    const { url, type } = media;
+    return nextProps.media.url !== url || nextProps.media.type !== type;
+  }
+
+  componentWillUnmount() {
+    const { playbackPos } = this.state;
+    const { userid, media, tab, hotReloadPlayback } = this.props;
+    ltikPromise.then(ltik => {
+      axios
+        .post(`/api/medias/playback?ltik=${ltik}`, {
+          userid,
+          file: media.file,
+          tab,
+          classShortname: media.classShortname,
+          time: playbackPos,
+        })
+        .then(() => {
+          if (tab === constants.TAB_DIGITAL_AUDIO_RESERVES) {
+            hotReloadPlayback(media.albumTitle, media.file, playbackPos);
+          }
+        })
+        .catch(err => {
+          console.log(err);
```

Comment:
ESLint error with console log

Reply:
This will be fixed in another ticket when I use axios retry.